core/files/css/base.php

```diff
@@ -471,6 +475,36 @@ public function parse_property_placeholder( array $control, $value, array $contr
 		return (string) $control_obj->get_style_value( $placeholder, $value, $control );
 	}
 
+	/**
+	 * Parse size units selectors dictionary.
+	 *
+	 * Check for placeholders in the controlSelector eg {{SIZE}} or {{UNIT}}
+	 * and replace it with the corresponding key's value from the control values.
+	 *
+	 * @since 3.13.0
+	 * @access public
+	 *
+	 * @param string $constrol_selector The selector to parse eg --e-con-grid-template-columns: {{SIZE}}{{UNIT}}
+	 * @param array  $controlValues  The control values eg { unit: 'fr', size: 2 }
+	 *
+	 * @return string The parsed value for the CSS eg --e-con-grid-template-columns: 2fr
+	 */
+	public function parse_size_units_selectors_dictionary( $constrol_selector, $control_values ) {
+		return preg_replace_callback( '/{{(.*?)}}/', function( $matches ) use ( $control_values ) {
```

Comment:
@louiswol94 

Can you please extract the regEx into a const for readability?

E.g. 
const selfExplanatoryVariableName = '/{{(.*?)}}/';

Reply:
@hein-obox Done